ltsp/common/ltsp/ltsp.conf

```diff
@@ -14,6 +14,16 @@
 # Specify an alternative TFTP_DIR
 # TFTP_DIR=/var/lib/tftpboot
 
+# Specify a CMDLINE_BOOT_METHOD for alternative boot methodes like RBD or iSCSI.
+# Here is an example RBD boot cmdline:
+# It follows this form: rbdroot=<mons>:<user>:<key>:<pool>:<image>[@<snapshot>]:[<partition>]:[<mountopts>]
+# Be aware that user is the username part only not "client.username" as often
+# referred to in Ceph. If the MON services listen on a non-standard port, port
+# shall be specified with ";" instead of standard ":" for obvious reasons -
+# e.g. host;7000
+# If you want to use the ${img} variable, you must use single quotes.
+# CMDLINE_BOOT_METHOD='boot=rbd ro rbdroot=10.101.0.4,10.101.0.5,10.101.0.6:user:secretkey==:rbd:${img}:1:'
+
```

Comment:
In the example ltsp.conf it would be best to include things that many people will need. I think RBD isn't too common, so I think it should be in the man page but not in the example ltsp.conf.

Reply:
Ceph/RBD, for its apparent benefits, will become increasingly popular, I believe. Ceph/RBD is already the defacto standard as an Openstack storage backend and gaining much popularity with Kubernetes as well.

There is a reason, why it is part of the mainline kernel and RedHat bought the Inkspace, the company developing it.

However, I believe it would not hurt to add an iSCSI example as well.